### PR TITLE
Upstream 6b774f2dfad7f8097b80539db30cfa27a355d299

### LINKED ISSUE #9377: Warning of disabled popups by window.open actions
relates to https://github.com/owncloud/web/pull/9159/commits/4ba7a443104df9a62f73905e578873cbe464bb3c

> Warning of disabled popups by window.open actions
> To avoid support tickets

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Fixes #9377

### CODE REVIEW

File: packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useGettext } from 'vue3-gettext'
+
+export const useBlockedPopupRedirectMessage = () => {
+  const { $gettext } = useGettext()
+
+  return {
+    title: $gettext('Blocked pop-ups and redirects'),
+    timeout: 20,
```
Comment (pascalwengerter) [inline packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts:8]: @elizavetaRa 10s felt too short, so I doubled it. Still quite a lot of text to digest for the average user though

**Comment by pascalwengerter** [comment]
Random CI failures..

**Review by dschmidt — COMMENTED** [review]
We were somehow not talking about the same thing... I meant, we should put the whole window open and message code into the composable - not just the message...

File: packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useGettext } from 'vue3-gettext'
+
+export const useBlockedPopupRedirectMessage = () => {
+  const { $gettext } = useGettext()
+
+  return {
+    title: $gettext('Blocked pop-ups and redirects'),
```
Comment (tbsbdr) [inline packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts:7]: Please use a slightly different title to make clear, that blocking is not a feature but that the app detected, that pop ups are getting blocked: 
**"Pop-up and redirect block detected"**

File: packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    timeout: 20,
+    status: 'warning',
+    desc: $gettext(
+      'Some features might not work correctly. Please enable pop-ups and redirects in Settings > Privacy & Security > Site Settings > Permissions'
```
Comment (tbsbdr) [inline packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts:11]: Please use a more generic message that applies to all browsers:
**"Please turn on pop-ups and redirects in your browser settings to make sure everything works right."**

**Review by dschmidt — APPROVED** [review]

**Review by dschmidt — APPROVED** [review]

**Pull request merged**